site/src/examples/date-picker/CustomFormatter.tsx

```diff
@@ -0,0 +1,10 @@
+import { ReactElement } from "react";
+import { DatePicker } from "@salt-ds/lab";
+
+export const CustomFormatter = (): ReactElement => (
+  <DatePicker
+    style={{ width: "200px" }}
+    dateFormatter={(date) => date?.toLocaleString() ?? ""}
```

Comment:
Very minor, this means different user would see different examples result. May not be easy to write doc against this.

Reply:
will follow up changing it to using the internationalized so it is always consistent